framework/src/wifi_manager/wifi_manager_msghandler.c

```diff
@@ -37,9 +37,11 @@ static handler_queue g_wifi_message_queue;
  * External functions
  */
 extern wifi_manager_result_e wifimgr_handle_request(wifimgr_msg_s *msg);
+#ifdef CONFIG_BUILD_FLAT
 #ifdef CONFIG_VIRTUAL_WLAN
 extern void vwifi_start(void);
 #endif
+#endif
 
```

Comment:
please move vwifi_start() call to apps/system/init/init.c
below 

#if defined(CONFIG_WIFI_MANAGER)
	(void)wifimgr_run_msghandler();
#endif
#ifdef CONFIG_VIRTUAL_WLAN
 	vwifi_start();
 #endif

Reply:
Done. 
But, this will still result in error during protected build. So, I have added BUILD_FLAT check for it. 
Please try to find a different method to call kernel api from app side, either with syscall or ioctl.